app/containers/Forms/EmissionGasFields.tsx

```diff
@@ -20,31 +39,41 @@ const EmissionGasFields: React.FunctionComponent<FieldProps> = ({
           </Col>
         </Col>
         <Col md={3}>
-          <Form.Control
-            type="number"
-            value={formData.annualEmission}
-            onChange={e => {
+          <registry.fields.NumberField
+            required
+            schema={annualEmissionSchema}
+            uiSchema={uiSchema.annualEmission}
+            formData={formData.annualEmission}
+            autofocus={autofocus}
+            idSchema={idSchema}
```

Comment:
```suggestion
            idSchema={idSchema.annualEmission as IdSchema}
```
Or something like this, look at the updated `ProductionFields`

Reply:
Updated